resources/docker-compose.yml

```
@@ -161,6 +183,47 @@ services:
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+
+  cobiv2:
+    image: ghcr.io/catalogfi/cobiv2:latest
+    container_name: cobiv2
+    depends_on:
+      - garden-evm-watcher
+    restart: unless-stopped
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)